source/install/desktop.rst

```diff
@@ -85,3 +85,6 @@ Desktop App window is black and doesn't load the page
     - Finally, try disabling GPU hardware acceleration by using the `--disable-gpu <http://peter.sh/experiments/chromium-command-line-switches/#disable-gpu>`_ Chromium command line switch.
 
     If none of the above steps resolve the issue, please open a new ticket in the `Mattermost Troubleshooting Forum <https://forum.mattermost.org/t/how-to-use-the-troubleshooting-forum/150>`_. 
+
+Desktop app appears off-screen after disconnecting from a secondary screen
+    Quit the app and delete the following file: ``USERPROFILE%\AppData\Roaming\Mattermost\bounds-info.json``
```

Comment:
Repro steps:

1-Open mattermost.
2-Move the window to a secondary screen.
3-Access the computer using the remote desktop feature as the same user.
4-Disconnect from the remote desktop session.
5-After logging in the computer directly, the mattermost window is still running but it spawns to an off-screen location on the secondary screen.

1-Open mattermost.
2-Move the window to a secondary monitor.
3-Move Mattermost to the secondary monitor.
4-Disconnect from the second monitor.
5-After logging in the computer directly, the mattermost window is still running but it spawns to an off-screen location on the secondary monitor.

Would you have a suggestion on how to better describe the issue? After re-reading I agree the `off-screen` isn't clear.

And yes, this is for the end user (since each user installs the app).